contributing/tests.md

```diff
@@ -7,268 +7,56 @@ describes them and gives pointers on how to run/debug them.
 
 The test types from most preferred to least preferred are:
 
-1. Unit Tests
-1. Vscode-integration Tests
-1. Integration Tests
-1. System Tests
+1. Unit Tests - jest: Found under the test/jest directory.
+   - `npm run test:unit`
+1. End to End Tests - _TDB_
 
-Ensure that you have set a default dev hub through the `sfdx force:auth:*` commands,
-passing in `--setdefaultdevhubusername`.
+**Deprecated**
 
-To run all tests, execute `npm run compile && npm run test` from the top-level
-folder.
+1. Unit Tests - mocha: Found under the test/unit directory. Not under active development.
+   - `npm run test:unit`
+1. vscode-integration - Tests that are somewhere between integration and unit tests. These are being phased out and there should be no new additions and minimual effort to
```

Comment:
effort to...?

Reply:
heh....I was debating `keep these running` but sort of thought better of it.  We're in a weird state until we get the e2e tests in place where we still need these tests. 

I think I'll just stop at no new additions.